src/app/calendar/_components/timetable/TimeTableWeekdayColumn.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+import { twMerge } from "tailwind-merge";
+import { ReservationWithTimestamp } from "./TimeTableMainPart";
+import { Reservation } from "../../page";
+import { useEffect, useState } from "react";
+
+type TimeTableDayContentProps = {
+  reservationTimeStamps: ReservationWithTimestamp[];
+  day: string;
+  typeFilters: string[];
+  selectedRoom: string;
+};
+
+export function TimeTableDayContent({
+  reservationTimeStamps,
+  day,
+  typeFilters,
+  selectedRoom,
+}: TimeTableDayContentProps) {
+  const roomFilters = ["2.41"];
+
+  const filteredReservations = filterReservationsWithProperTypeAndRoom(
+    reservationTimeStamps,
+    typeFilters,
+    selectedRoom,
+  );
+
+  function filterReservationsWithProperTypeAndRoom(
+    reservationTimeStamps: ReservationWithTimestamp[],
+    typeFilters: string[],
+    roomFilter: string,
+  ) {
+    return reservationTimeStamps.map(
+      (reservation: ReservationWithTimestamp) => {
```

Comment:
you don't have to put typing here, if you do that means you don't have correct type on `reservationTimeStamps` but I believe you do
```suggestion
      (reservation) => {
```